sma-em-dev/config.yaml

```diff
@@ -28,6 +29,7 @@ options:
     RECONNECT_INTERVAL: 86400
     DEBUG: 0
 schema:
+    MCASTGRP: str
```

Comment:
Would it make sense to have this schema as `str?` 

And then have the default in python options dict (instead of “”)

then I might not have to add it to my existing install